server/images.go

```diff
@@ -340,7 +340,24 @@ func CreateModel(ctx context.Context, name, modelFileDir, quantization string, m
 					return err
 				}
 			} else if strings.HasPrefix(c.Args, "@") {
-				blobpath, err := GetBlobsPath(strings.TrimPrefix(c.Args, "@"))
+				digest := strings.TrimPrefix(c.Args, "@")
+				if ib, ok := intermediateBlobs.Load(digest); ok {
```

Comment:
aren't `ib` and `digest` the same thing? Can't you just call this as `_, ok := intermediateBlobs.Load(digest); ok {` and use `digest` below?